src/openproduct/producten/tests/api/test_product_filters.py

```diff
@@ -851,3 +851,18 @@ def test_eigenaar_vestigingsnummer_filter(self):
                 response.data["results"][0]["eigenaren"][0]["vestigingsnummer"],
                 "12345678",
             )
+
+    def test_filter_by_naam(self):
```

Comment:
```suggestion
    def test_naam_filter(self):
```
small nitpick but lets keep the naming it consistent.